src/github3/pulls.py

```diff
@@ -575,6 +576,15 @@ def reviews(self, number=-1, etag=None):
         url = self._build_url('reviews', base_url=self._api)
         return self._iter(int(number), url, PullReview, etag=etag)
 
+    def combined_status(self):
+        """ Retrieve the combined status for this Pull Request
+        :rtype:
+            :class:`~github3.repos.CombinedStatus`
+        """
+        combined_status_url = self.statuses_url.replace('/statuses/', '/status/')
```

Comment:
This feels a bit clumsy, but I can't immediately think of a better way to do this. Perhaps another reviewer will have a suggestion.

Reply:
I was also struggling to find a neater way :/ i don't see any other place we actually build urls so i had nothing to go off

Reply:
What does `statuses_url` look like (when the API provides it)?

Reply:
you can find it here: https://developer.github.com/v3/pulls/
`"statuses_url": "https://api.github.com/repos/octocat/Hello-World/statuses/6dcb09b5b57875f334f61aebed695e2e4193db5e",`